source/core/ut_suite_manager.pkb

```diff
@@ -659,10 +687,40 @@ create or replace package body ut_suite_manager is
     return l_suites;
   end;
 
+  function build_no_test_error_msg(
+    a_schema_name in varchar2,
+    a_suite_path in varchar2,
+    a_procedure_name in varchar2,
+    a_object_name in varchar2,
+    a_tags in ut_varchar2_rows := ut_varchar2_rows()  
+    ) return varchar2 is
+    l_error_msg varchar2(500);
+    l_tags clob:= ut_utils.table_to_clob(coalesce(a_tags,ut_varchar2_rows()),',');
+  begin
+    if a_suite_path is not null then 
+      l_error_msg := 'No suite packages found for path '||a_schema_name||':'||a_suite_path;
+    elsif a_procedure_name is not null then 
+      l_error_msg := 'Suite test '||a_schema_name||'.'||a_object_name|| '.'||a_procedure_name||' does not exists';
+    elsif a_object_name is not null then
+      l_error_msg := 'Suite package '||a_schema_name||'.'||a_object_name|| ' does not exists';
+    end if;
+    
+    if l_error_msg is null and a_tags.count > 0 then
```

Comment:
I would refrain from raising an error in this case.
If we pass 3 paths, and 1 tak and one of paths is not tagged, I would expect utPLSQL to execute the tagged items from remaining two paths.

If given suite doesn't have any of given tags it is OK, we should not throw exception, just run 0 (zero) tests.